tests/db_migrator_input/config_db/reclaiming-buffer-traditional-single-pool-expected.json

```diff
@@ -0,0 +1,343 @@
+{
+    "BUFFER_PG|Ethernet0|0": {
+        "profile": "ingress_lossy_pg_zero_profile"
+    },
+    "BUFFER_PG|Ethernet4|0": {
+        "profile": "ingress_lossy_pg_zero_profile"
+    },
+    "BUFFER_PG|Ethernet4|3-4": {
+        "profile": "pg_lossless_10000_5m_profile"
+    },
+    "BUFFER_PG|Ethernet8|3-4": {
```

Comment:
zero profile for Ethernet8?

Reply:
Ethernet8's lossless profile doesn't match its speed. So it won't be treated as the default configuration and zero profile won't be applied.

|port|speed|cable length|lossless pg|lossy pg|default|
|:---:|:---:|:---:|:---:|:---:|:---|
|ethernet0|||None|none|yes|
|ethernet4|10000|5m|pg_lossless_10000_5m_profile|none|yes|
|ethernet8|25000|5m|pg_lossless_10000_5m_profile|none|no|
|ethernet12||||exist|no|
|ethernet16||||exist|no|
|ethernet20|100000|5m|pg_lossless_100000_5m_profile|none|yes|
|ethernet24|||none|none|yes|